examples/example_server.py

```diff
@@ -41,6 +42,45 @@ class Foo(enum.Enum):
     GHI_K = 2
 
 
+class Total(aiokatcp.AggregateSensor):
+    def __init__(self, target):
+        super().__init__(target=target, sensor_type=int, name="total")
+
+    def update_aggregate(self, updated_sensor, reading, old_reading):
+        if updated_sensor is None:
+            # Instantiation, calculate total for sensors already in target.
+            total = sum(
+                sensor.value for sensor in self.target.values() if self.filter_aggregate(sensor)
+            )
+            return aiokatcp.Reading(time.time(), aiokatcp.Sensor.Status.NOMINAL, total)
+        if reading is None:
+            # The sensor is being removed from the set.
+            return aiokatcp.Reading(
+                updated_sensor.timestamp,
+                aiokatcp.Sensor.Status.NOMINAL,
+                self.value - updated_sensor.value,
+            )
+        if old_reading is None:
+            # The sensor is being added to the set.
+            return aiokatcp.Reading(
+                updated_sensor.timestamp,
+                aiokatcp.Sensor.Status.NOMINAL,
+                self.value + updated_sensor.value,
+            )
+        # Otherwise, it's just a change.
+        return aiokatcp.Reading(
+            updated_sensor.timestamp,
+            aiokatcp.Sensor.Status.NOMINAL,
+            self.value - old_reading.value + reading.value,
+        )
```

Comment:
This can probably be simplified/shortened somewhat. I haven't tried running it, but how about something like
```suggestion
        new_value = self.value
        if old_reading is not None:  # Will be None if this is a new sensor being added
            new_value -= old_reading.value  # Remove the previous value from the sum
        if reading is not None:  # Will be None if this is a sensor being removed
            new_value += reading.value  # Add the new value to the sum
        return aiokatcp.Reading(
            updated_sensor.timestamp,
            aiokatcp.Sensor.Status.NOMINAL,
            new_value,
        )
```